src/app/modules/core/dialogs/debug-information/debug-information.dialog.ts

```diff
@@ -25,9 +25,9 @@ export class DebugInformationDialog {
     // @ts-ignore
     this.os = navigator.userAgentData?.platform ?? translate.instant("UNKNOWN");
     // @ts-ignore
-    this.browser = navigator.userAgentData?.brands[0].brand ?? translate.instant("UNKNOWN");
+    this.browser = navigator.userAgentData?.brands[1].brand ?? translate.instant("UNKNOWN");
```

Comment:
user agent wordt door de browser meegestuurd, wat als er geen 2e "brand" in zit? of helemaal geen brand?